block-storage-attacher/README.md

```diff
@@ -1,5 +1,5 @@
 ![Build Status](https://travis-ci.com/IBM/ibmcloud-storage-utilities.svg?branch=master)
-[![Coverage](https://pages.github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/coverage/master/badge.svg)](https://pages.github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/coverage/master/cover.html)
+[![Coverage](https://github.com/IBM/ibmcloud-storage-utilities/blob/gh-pages/block-storage-attacher/coverage/master/badge.svg)](https://pages.github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/coverage/master/cover.html)
```

Comment:
You have changed only the path of badge.svg file. Even cover.html path needs change.
And the path should be `https://github.com/IBM/ibmcloud-storage-utilities/tree/gh-pages/block-storage-attacher/coverage/master` I believe as thats what I see in browser.

Reply:
Ok I will change actually the the file path we provide for cover.html is just reference only